docs/configurations/_index.md

```diff
@@ -0,0 +1,233 @@
+---
+bookIconFa: fa-gears
+bookFlatSection: true
+url: configurations
+weight: 470
+---
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# Configurations
```

Comment:
Isn't this added automatically?

Reply:
This isn't but that's a great idea to add that feature to the iceberg-theme. It should be as simple as adding this HTML below to the [body.html](https://github.com/apache/iceberg-docs/blob/main/iceberg-theme/layouts/partials/body.html#L25) template right above `{{- .Content -}}`. Let me test that out!
```html
<h1 id="{{ lower .Title }}">{{ .Title }}<a href="#{{ lower .Title }}" class="anchortag" arialabel="Anchor"> 🔗 </a></h1>
```